user_guide_src/source/changelogs/v4.4.2.rst

```diff
@@ -24,6 +24,8 @@ Deprecations
 Bugs Fixed
 **********
 
+- **Security:** Fixed a bug where the CSRF token wasn't checked if we sent it in the raw body (not JSON format) for PUT, PATCH, and DELETE requests.
```

Comment:
I think this is not a bug fix.

It may be hard to find, but it states that only JSON requests are checked:
https://codeigniter4.github.io/CodeIgniter4/libraries/security.html#the-order-of-token-sent-by-users

Reply:
Good point. I have moved this to the `Changes` category in the changelog and updated the user guide.